editor/editor_spin_slider.cpp

```diff
@@ -371,17 +371,24 @@ void EditorSpinSlider::_draw_spin_slider() {
 			grabber->hide();
 		}
 	} else if (!hide_slider) {
-		int grabber_w = 4 * EDSCALE;
-		int width = size.width - sb->get_minimum_size().width - grabber_w;
-		int ofs = sb->get_offset().x;
-		int svofs = (size.height + vofs) / 2 - 1;
+		const int grabber_w = 4 * EDSCALE;
+		const int width = size.width - sb->get_minimum_size().width - grabber_w;
+		const int ofs = sb->get_offset().x;
+		const int svofs = (size.height + vofs) / 2 - 1;
 		Color c = fc;
-		c.a = 0.2;
 
+		// Draw the horizontal slider's background.
+		c.a = 0.2;
 		draw_rect(Rect2(ofs, svofs + 1, width, 2 * EDSCALE), c);
-		int gofs = get_as_ratio() * width;
+
+		// Draw the horizontal slider's filled part on the left.
+		const int gofs = get_as_ratio() * width;
+		c.a = 0.45;
+		draw_rect(Rect2(ofs, svofs + 1, gofs, 2 * EDSCALE), c);
```

Comment:
Should there be a case when we just don't draw anything if the ratio is 0?

Reply:
The appearance of sliders with a ratio of 0 seems identical before and after this PR, so I don't think this is needed.

### Before

![2021-12-10_17 31 23](https://user-images.githubusercontent.com/180032/145608342-1a821342-b9cf-4b38-a8a0-24a60d17ddd5.png)

### After

![2021-12-10_17 32 02](https://user-images.githubusercontent.com/180032/145608362-af285b1b-5d41-41cc-b5a7-21d3e635015c.png)